app/views/users/new.html.haml

```diff
@@ -0,0 +1,25 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      =bootstrap_form_for @user, class: "form-horizontal" do |f|
+        =f.text_field :username
+        =f.email_field :email
+        =f.password_field :password
+        =f.submit "Sign Up", class: 'btn btn-default'
+/      %form.form-horizontal
```

Comment:
It is even more important to cleanup commented code in your template views. This code is rendered as html comments and can be viewed by anybody using browser dev tools.